egs/madcat_ar/v1/local/apply_bpe.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env python
```

Comment:
Let's consider putting these BPE related scripts in steps/bpe/.  @hhadian, do you have any thoughts on the location?

Reply:
Maybe `utils/lang/bpe` because it's related to dict/lang preparation?